### PR TITLE
Oregon City cleanup

### PR DESCRIPTION
**Pull request by liamjwang**

### CODE REVIEW

**Comment by liamjwang** [comment]
Basic functionality is tested-- ideally, we would check more thoroughly to make sure everything is working.

**Comment by RobinsonZ** [comment]
Have we made any major semantic changes aside from the PointDrive et al inheritance structure?

**Comment by liamjwang** [comment]
I don't think so (as far as I can remember).

File: src/main/java/org/team1540/robot2019/Robot.java [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+//        tebPlanner = new TEBPlanner(() -> new Odometry(odometry.getOdomToBaseLink(), drivetrain.getTwist()), 5801, 5800,
+//            "10.15.40.202", 0.01);
+
+        OI.init();
+
+        ShuffleboardDisplay.init();
+
+        // TODO: use shuffleboard properly
+        SmartDashboard.putBoolean("IsHatchPreload", false);
```
[the remark this answers is not in the document]
Comment (liamjwang) [reply src/main/java/org/team1540/robot2019/Robot.java]: I don't think so? command-shift-f

File: src/main/java/org/team1540/robot2019/Robot.java [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        SmartDashboard.setDefaultBoolean("EnableBackupCam", false);
+        if (SmartDashboard.getBoolean("EnableBackupCam", false)) {
+            UsbCamera cam = CameraServer.getInstance().startAutomaticCapture("backup cam", 0);
+            cam.setResolution(128, 73);
+            cam.setFPS(30);
+        }
+
+        SmartDashboard.putNumber("CalibrationDistance", 1);
+        Command estimatePitch = new SimpleCommand("Estimate Camera Pitch", () -> {
+            Double calibrationPitch = deepSpaceVisionTargetLocalization.estimateCorrectPitch(SmartDashboard.getNumber("CalibrationDistance", 0), 1000, 0.001);
```
[the remark this answers is not in the document]
Comment (liamjwang) [reply src/main/java/org/team1540/robot2019/Robot.java]: yeah, the calibration wasn't meant to be in this branch... should be removed

File: src/main/java/org/team1540/robot2019/Robot.java [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+            drivetrain::getRightPositionMeters,
+            Hardware.navx::getAngleRadians,
+            0.01
+        );
+
+        Hardware.limelight = new Limelight("limelight-a",
+            new Transform3D(RobotMap.CAM_X, RobotMap.CAM_Y, RobotMap.CAM_Z, RobotMap.CAM_ROLL,
+                RobotMap.CAM_PITCH, RobotMap.CAM_YAW));
+        lastOdomToVisionTargetTracker = new LastValidTransformTracker(odometry::getOdomToBaseLink);
+        deepSpaceVisionTargetLocalization = new DeepSpaceVisionTargetLocalization(Hardware.limelight,
```
[the remark this answers is not in the document]
Comment (liamjwang) [reply src/main/java/org/team1540/robot2019/Robot.java]: No

**Pull request merged**